extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -545,6 +559,21 @@ export class AmpStoryPage extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Rewinds all media on this page.
+   * @return {!Promise} Promise that resolves after the callbacks are called.
+   * @private
+   */
+  rewindAllMedia_() {
+    return this.whenAllMediaElements_((mediaPool, mediaEl) => {
+      if (this.isBotUserAgent_) {
+        // No-op.
```

Comment:
We still need to rewind the video here. We should match our [rewind code](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/media-tasks.js#L306) in the media pool, and do `mediaEl.currentTime = 0;`